types/audit-details.d.ts

```diff
@@ -193,6 +193,7 @@ declare global {
         type: 'node';
         path?: string;
         selector?: string;
+        boundingRect?: Artifacts.Rect;
```

Comment:
I'd love to make all these required sometime and see what else we need to fix :)